pyproject.toml

```diff
@@ -17,4 +85,4 @@ reportUnnecessaryTypeIgnoreComment = true
 version = "literal: pygitguardian/__init__.py: __version__"
 format = "md"
 md_header_level = "2"
-insert_marker = "# Changelog"
+insert_marker = "# Changelog"
```

Comment:
Looks like your editor is not configured to write trailing new lines (same thing for setup.cfg).